draft-ietf-mls-protocol.md

```diff
@@ -2891,7 +2891,7 @@ values have been consumed and MUST be deleted:
 Concretely, suppose we have the following Secret Tree and ratchet for
 participant D:
 
-~~~ ascii-art
+~~~ aasvg
```

Comment:
```suggestion
~~~ ascii-art
```

This one doesn't quite work either; the bottom lines of the key/nonce forks don't render properly.